akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/ReplayingSnapshot.scala

```diff
@@ -153,13 +153,15 @@ private[akka] class ReplayingSnapshot[C, E, S](override val setup: BehaviorSetup
           state = setup.snapshotAdapter.fromJournal(snapshot)
           setup.internalLogger.debug("Loaded snapshot with metadata [{}]", metadata)
           metadata.metadata match {
-            case Some(rm: ReplicatedSnapshotMetadata) => (metadata.sequenceNr, rm.seenPerReplica, rm.version)
```

Comment:
No default here, so the consuming stream looking up each replica with `GetSeenSequenceNr` during per replica replication stream startup would fail.